Core/Src/main.c

```diff
@@ -66,8 +66,7 @@ static void MX_CRC_Init(void);
 
 /* Private user code ---------------------------------------------------------*/
 /* USER CODE BEGIN 0 */
-//volatile uint8_t TxData[2] = {0};
-volatile uint32_t TxData[3];
+uint32_t TxData[3];
```

Comment:
Why is `volatile` removed?

Reply:
I didn't mean to remove that part. Originally I had it pass into one of the transmit functions so I was testing if it was needed but forgot to change it back. Thanks for catching that!